tasty-plutus/src/Test/Tasty/Plutus/Internal/Feedback.hs

```diff
@@ -45,6 +53,51 @@ import Prelude hiding ((<>))
 ourStyle :: Style
 ourStyle = style {lineLength = 80}
 
+explainFailureExhaustion :: String
+explainFailureExhaustion =
+  renderStyle ourStyle $
+    "Gave up trying to generate a successful run for estimation."
+      $+$ "This can happen due to what \'failure\' means for a script in tasty-plutus:"
+      $+$ hang " * " 4 "The script ran successfully, but didn't log a success from its wrapper; or"
+      $+$ hang " * " 4 "The script crashed or failed to evaluate."
+      $+$ "Because of how the CEK evaluator for scripts works, we can only establish an estimate for the first case."
+      $+$ "As property tests generate inputs pseudorandomly, even tests where failure is not guaranteed, if success isn't likely enough, we might never see an instrumentable case."
+      $+$ "If you are seeing this message, this is exactly what happened."
```

Comment:
I think that this verbose style is quite overwhelming and cluttering the test report, especially if someone has a whole bunch of tests which are expected to fail. I'd argue that that's a quite common case. Could we perhaps state a one-liner "Cannot provide estimates for test cases" instead and provide the more detailed explanation at the end of the tests report? That would make the report more readable.

Reply:
I've modified the code that does the reporting to give a briefer message, and instead documented the 'failure problem' more extensively in the Haddocks for PlutusEstimate.